src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -363,6 +361,10 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	}
 
 	private func riskCell(forRowAt indexPath: IndexPath) -> UITableViewCell {
+		if let riskCell = riskCell {
```

Comment:
Wouldn't this prevent the risk-cell to update with a new model and a new risk-state?

Reply:
The risk cell is also updated from the HomeState, that's why we only need to set it up once.